### ISSUE TITLE
German Translation

### ISSUE THREAD

**Issue by Dlurak** [JonasCz/How-To-Prevent-Scraping#10]
Would it be OK to translate the English version to a German one?
After all there is also a Chinese version.

**Comment by JonasCz** [comment]
Yes of course!, you can also add a link to your translation afterwards. Keep in mind that this information is a bit obsolete by now, it was written in 2017.
